test.disabled/ejabberd_tests/tests/mam_SUITE.erl

```diff
@@ -1354,6 +1377,42 @@ archived(Config) ->
     %% Made fresh in init_per_testcase
     escalus:story(Config, [{alice, 1}, {bob, 1}], F).
 
+
+message_with_stanzaid_and_archived(Config) ->
+    F = fun(Alice, Bob) ->
+        %% Archive must be empty.
+        %% Alice sends "OH, HAI!" to Bob.
+        escalus:send(Alice, escalus_stanza:chat_to(Bob, <<"OH, HAI!">>)),
+
+        %% Bob receives a message.
+        Msg = escalus:wait_for_stanza(Bob),
+        try
```

Comment:
We usually don't use `try..catch` in tests. Let it crash. :) Common Test will print error information nicely anyway.